src/App.tsx

```diff
@@ -10,33 +12,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
+  const [selectedTab, setSelectedTab] = useState<Tab>(tabs[0]);
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const onTabSelected = (chosenTab: Tab) => {
+    // console.log(chosenTab);
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+    // const currentTab = tabs.find(tab => tab.id === chosenTab.id) || tabs[0];
 
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
+    setSelectedTab(chosenTab);
+  };
 
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${selectedTab.title}`}
+      </h1>
+      <Tabs
+        tabs={tabs}
+        selectedTab={selectedTab}
+        onTabSelected={onTabSelected}
```

Comment:
```suggestion
        onTabSelected={handleTabSelected}
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364
Also, I don't see you passing the selectedTabId
![image](https://user-images.githubusercontent.com/96409563/212009753-f28b7fa0-3254-4ad5-b4ba-148df599106d.png)